src/V3ExecGraph.cpp

```diff
@@ -109,52 +110,122 @@ class ThreadSchedule final {
 
         // Thread labels
         *logp << "\n  // Threads\n";
-        const int threadBoxWidth = 2;
-        for (int i = 0; i < v3Global.opt.threads(); i++) {
-            *logp << "  t" << i << " [label=\"Thread " << i << "\" width=" << threadBoxWidth
-                  << " pos=\"" << (-threadBoxWidth / 2) << "," << -i
-                  << "!\" style=\"filled\" fillcolor=\"grey\"] \n";
+
+        const auto emitBlock
+            = [&](const string& name, const string& label, double width, double height,
+                  double xPos, double yPos, const string& fillColor) {
+                  *logp << "  " << name << " [label=\"" << label << "\" width=" << width
+                        << " height=" << height << " pos=\"" << xPos << "," << yPos
+                        << "!\" style=\"filled\" fillcolor=\"" << fillColor << "\"]\n";
+              };
```

Comment:
Please make into a "normal" function.  Similar for most (if not all - unsure) lambdas below.